apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector/preview.scss

```diff
@@ -0,0 +1,189 @@
+@import '~@wordpress/base-styles/variables';
+@import './shared.scss';
+
+// Preview positioning
+$preview-max-width: 960px;
+$preview-right-margin: 24px;
+$preview-left-sidebar-reduced: calc( 30% + #{$admin-sidebar-width-collapsed} );
+$preview-left-sidebar-full: calc( 30% + #{$admin-sidebar-width} );
+
+$template-large-preview-title-height: 120px;
+
+// Template Selector Preview
+.template-selector-preview {
+	@media screen and ( max-width: 659px ) {
```

Comment:
Do we have mixins or variables available in Gutenberg for these screen sizes?

Reply:
Couldn't find any (tried grepping `660px` and `659px`).

Reply:
Gotcha — I guess in some follow up some day we could re-use breakpoints from `~@wordpress/base-styles/breakpoints`